pkg/oci/utils.go

```diff
@@ -33,8 +33,8 @@ var (
 	// ErrNoLinux is an error for missing Linux sections in the OCI configuration file.
 	ErrNoLinux = errors.New("missing Linux section")
 
-	// ConfigPathKey is the annotation key to fetch the OCI configuration file path.
-	ConfigPathKey = "com.github.containers.virtcontainers.pkg.oci.config_path"
+	// ConfigJSONKey is the annotation key to fetch the OCI configuration.
+	ConfigJSONKey = "com.github.containers.virtcontainers.pkg.oci.config"
```

Comment:
So by saving the entire JSON, we protect ourselves against the case where the failure happens before we could store the config.json on disk ?

Reply:
We protect ourselves by not relaying in the config.json from the bundle in oci. The runtime would need to save all it needs in its state file. In this case the state is the vircontainers state file and the runtime perform some operations using the information provides in this json.